### PR TITLE
chore: reduce insertion size of fuzz test

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

Reduce data insertion size of `fuzz_insert` & `fuzz_insert_logical_table` targets.

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

## Summary by CodeRabbit

- **New Features**
  - Enhanced fuzz testing customization by allowing maximum values for columns, rows, tables, and actions to be set via environment variables.
  
- **Refactor**
  - Updated multiple fuzz testing scripts to utilize a new utility function for retrieving values from environment variables, improving flexibility and consistency.

- **Chores**
  - Added new constants and utility functions to support environment variable-based configurations.

### CODE REVIEW

**Comment by fengjiachun** [comment]
Could you explain why we need to reduce the size of data insertion? @WenyXu

**Comment by killme2008** [comment]
It's better to be configured by environment variables, and we can configure it larger in release ci and a smaller value for every PR.

**Comment by killme2008** [comment]
> Could you explain why we need to reduce the size of data insertion? @WenyXu

Just want to speed up the CI.

**Comment by WenyXu** [comment]
> It's better to be configured by environment variables, and we can configure it larger in release ci and a smaller value for every PR.

Good point, I'm going to add this feature.

**Review by fengjiachun — APPROVED** [review]

File: tests-fuzz/targets/fuzz_alter_table.rs [hunk 1]
```diff
@@ -67,7 +70,8 @@ enum AlterTableOption {
 }
 
 fn generate_create_table_expr<R: Rng + 'static>(rng: &mut R) -> Result<CreateTableExpr> {
-    let columns = rng.gen_range(2..30);
+    let max_columns = get_from_env_or_default_value(GT_FUZZ_INPUT_MAX_COLUMNS, 30);
+    let columns = rng.gen_range(1..max_columns);
```
Comment (evenyag) [inline tests-fuzz/targets/fuzz_alter_table.rs:74]: Here we change it from `2` to `1`, is it expected?

**Pull request merged**